openmdao/docs/features/core_features/working_with_derivatives/simul_derivs.rst

```diff
@@ -176,6 +176,19 @@ option as follows:
 
 which will display a visualization of the sparsity
 structure with rows and columns labelled with the response and design variable names, respectively.
+See the figure :ref:`here <fig-coloring-view>`.
+
+
+A text-based view is also available using the `--jtext` arg.  For example:
+
+
+.. code-block:: none
+
+    openmdao total_coloring <your_script_name> --jtext
+
```

Comment:
Anyway to also make this work from inside your run script, much like `view_model(prob)`?

Reply:
Programmatically you can call display() on the Coloring object.  The coloring object is accessible via system._coloring_info['coloring'] or can be loaded from a file using Coloring.load(fname)